dxr/plugins/descriptor/tests/test_descriptions/test_descriptions.py

```diff
@@ -0,0 +1,37 @@
+"""Test that the plugin finds the expected descriptions, and they appear on the
+browse pages.
+
+"""
+from nose.tools import ok_
+
+from dxr.testing import DxrInstanceTestCase
+
+
+class DescriptionTests(DxrInstanceTestCase):
+    def test_browse_page(self):
+        """Test that the expected descriptions appear on the root page.
+        """
```

Comment:
If you're going to skip a line, it goes inside the quotes (for some odd historical emacs reason in PEP0008 that has since been deleted, but it's consistent with the rest of the codebase).